### ISSUE TITLE
Go to definition opens empty dropdown box and immediately closes it again

### ISSUE THREAD

**Issue by narpfel** [sadovnychyi/autocomplete-python#235]
This leads to an annoying flickering.

[It seems](https://github.com/sadovnychyi/autocomplete-python/blob/aa75abf71b1ca923888e3f53cc2876263769c368/lib/provider.coffee#L516-L520) that the `DefinitionsView` is not really needed if `results.length == 1` . Would it be possible to skip constructing it in the first place so that the flickering is prevented?

Thanks in advance!

**Comment by sadovnychyi** [comment]
Hm, I've never seen it, beside that "Looking for definition" placeholder which is a loading screen. It's probably depends on how fast Atom is running.. I'm not sure what we could skip here if we have to build that view anyway to show that loading screen.

**Comment by narpfel** [comment]
Yeah, the loading screen is what I mean. I didn’t even realize it was a loading screen because it vanished so quickly.

The question is if the loading screen is needed if it’s only visible for like a tenth of a second. Maybe something less obtrusive like an entry in the status bar would also work?

**Comment by sadovnychyi** [comment]
It seems more complicated to show different load screens depending on the time. And for big projects it could take seconds to find definitions if there are no cache.

Maybe it's possible to remove that text entry field above "Looking for definition" to make it look better.
